node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const note = process.argv;
+let data = '';
+for (var i = 2; i < note.length; i++) {
+  const noteArray = note[i];
```

Comment:
Even though this code will get thrown away :-) I will comment that it feels like you used `note` and `noteArray` backwards. Your `note` variable holds an array while your `noteArray` variable holds just one word of the note.

Reply:
so true... by line 6 i realized i had gone too far and couldn't go back and must see my mistakes through to the bitter end